packages/web-frontend/src/containers/DashboardExportModal.js

```diff
@@ -34,9 +37,23 @@ const DialogContent = styled(BaseDialogContent)`
   }
 `;
 
-const MuiButton = styled(MuiIconButton)`
-  margin: 0px 20px;
+const PrimaryButton = styled(BasePrimaryButton)`
+  text-transform: capitalize;
+  margin-left: 1rem;
+`;
+const TextButton = styled(Button)`
```

Comment:
It would be good to add some more button components to the Buttons file at some point and update the MUITheme to be a bit more useful. I'll add some points to my refactor notes.